ikarus/finiteelements/mechanics/kirchhoffloveshell.hh

```diff
@@ -70,7 +69,7 @@ namespace Ikarus {
       auto& first_child = this->localView().tree().child(0);
       const auto& fe    = first_child.finiteElement();
       numberOfNodes     = fe.size();
-      dispAtNodes.resize(fe.size());
+      dispAtNodes.resize(numberOfNodes);
```

Comment:
This shouldn't be here maybe, this could be a small different PR

Reply:
I ment the changes in the KL shell in general. Also this fix of the bracket is no disguised in this PR.